exec/java-exec/src/test/java/org/apache/drill/exec/metastore/TestMetastoreWithEasyFormatPlugin.java

```diff
@@ -1079,14 +1079,14 @@ public void testFilesPruningWithLimit() throws Exception {
       queryBuilder()
           .sql("select * from dfs.tmp.`%s` limit 1", tableName)
           .planMatcher()
-          .include("Limit", "numFiles=1,")
+          .include("Limit", "numFiles=12,")
```

Comment:
This is the regression I mentioned in the previous comment. It is non-optimal to read all 12 files instead of the single one.

Reply:
Reverted test to orginal